app/src/main/java/org/listenbrainz/android/ui/screens/yim23/Yim23FriendsScreen.kt

```diff
@@ -92,7 +93,7 @@ private fun Yim23Friends (viewModel: Yim23ViewModel) {
                     color = MaterialTheme.colorScheme.background
                 , onClick = {
                         try {
-                            uriHandler.openUri("https://beta.listenbrainz.org/user/${followers.value?.data?.followers!![page]}/year-in-music/2023/")
+                            uriHandler.openUri(Constants.LB_BASE_URL + "user/${followers.value?.data?.followers!![page]}/year-in-music/2023/")
```

Comment:
Try avoiding the force unwraps in the code. This could lead to unexpected crashes